README.md

```diff
@@ -10,8 +10,44 @@ To configure the repository and run tests, execute `npm run all`.
 
 ## Requirements
 
-The `aura-pos` branch should be checked out in `../parity-ethereum` and built in
-release mode.
+To integrate with [parity-ethereum](https://github.com/poanetwork/parity-ethereum), the following structure of folders is assumed:
+```
+.
+├── parity-ethereum
+├── posdao-test-setup
+```
+So there should be two folders on the same level and `posdao-test-setup` will use parity binary from the `parity-ethereum` folder, namely the binary is assumed to be at `../parity-ethereum/target/release/parity` relative to `posdao-test-setup` root.
+
+If you are working on modifications of `parity-ethereum` or want to compile specific branch/version, you can clone it directly and build the binary
+```bash
+# move up from posdao-test-setup root
+cd ..
+# clone over https
+git clone -b aura-pos https://github.com/poanetwork/parity-ethereum.git
+# OR over ssh
+git clone -b aura-pos git@github.com:poanetwork/parity-ethereum.git
+cd parity-ethereum
+# assumes you have Rust (>= 1.34) and Cargo installed
```

Comment:
I believe you that it is true that 1.34 is the minimum version but why wouldn't ask the user to get the latest stable version? That might be more future-proof. That is, they should do
```
rustup override set stable
```
in the `parity-ethereum` directory.